CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#332](https://github.com/nf-core/taxprofiler/pull/332) Improved meta map stability for more robust pipeline resuming (fix by @jfy133)
 - [#338](https://github.com/nf-core/taxprofiler/pull/338) Fixed wrong file 'out' file going to `centrifuge kreport` module (♥ to @LilyAnderssonLee for reporting, fix by @jfy133)
 - [#342](https://github.com/nf-core/taxprofiler/pull/342) Fixed docs/usage to correctly list the required database files for Bracken and tips to obtain Kraken2 databases (fix by @husensofteng)
+- [#351](https://github.com/nf-core/taxprofiler/issues/351) Add combineProfilesWithDatabase to enhance profiler `ganon` (fix by @LilyAnderssonLee)
```

Comment:
Actually this is the same as 307. Could you add yourself and also @Midnighter to that entry in teh changelog?

Reply:
Sure!